modules/administration/pages/backup-restore.adoc

```diff
@@ -13,6 +13,10 @@ Regardless of the backup method you use, you must have available at least three
 Running out of space can result in backups failing, so check this often.
 ====
 
+The following section describes backing up data within the {productname} container.
+On the container host, use [literal]``mgrctl term`` to open a terminal in the container before you execute commands.
```

Comment:
I think not all the commands are to be runned inside the container. So we should be careful with this sentence in here.

Reply:
Yes, you are right., we'd better remove that statement here.  All the procedures in the following sections are now adjusted as needed.